### PR TITLE
Windows wsl troubleshooting

### PR DESCRIPTION
**Pull request by TomHKeysight**
Removed the changes to the code, leaving only an update to the documentation

### CODE REVIEW

File: src/kyber512/indcpa.c [hunk 1]
```diff
@@ -28,10 +28,11 @@ static void matacc(poly* r, polyvec *b, unsigned char i, const unsigned char *se
   unsigned int ctr, pos, k, l;
   uint16_t val0, val1;
   int16_t c[4];
+  int j;
```
Comment (rwols) [inline src/kyber512/indcpa.c]: Why did you change these files?

**Review by rwols — CHANGES_REQUESTED** [review]
Don't change indcpa.c and fips202.c please

File: src/kyber512/indcpa.c (hunk 1 above)
Comment (TomHKeysight) [reply src/kyber512/indcpa.c]: Because this way of declaring integers inside of for loops is a feature ofC-99, which the windows compiler we recommend cannot compile correctly while also compiling the assembly statements we use.

Comment (TomHKeysight) [reply src/kyber512/indcpa.c]: However, since we'll stop recommending that people compile the pinata firmware on windows (without wsl) all together in the next pinata documentation, I'm reverting this change. It is not needed.

**Review by rwols — APPROVED** [review]

File: src/kyber512/indcpa.c (hunk 1 above)
Comment (rwols) [reply src/kyber512/indcpa.c]: But this code is not meant to be compiled with MSVC.
